packages/create-cloudflare/src/deploy.ts

```diff
@@ -99,24 +103,42 @@ export const runDeploy = async (ctx: C3Context) => {
 			: []),
 	];
 
-	const result = await runCommand(deployCmd, {
-		silent: true,
+	const outputFile = join(
+		await mkdtemp(join(tmpdir(), "c3-wrangler-deploy-")),
+		"output.json",
+	);
+
+	await runCommand(deployCmd, {
```

Comment:
is removing that `silent; true` flag that switches the command from non-interactive to interactive?